nodes/usb_cam_node.cpp

```diff
@@ -162,6 +169,8 @@ class UsbCamNode
     node_.param("gamma_max", gamma_max_, 200);
     node_.param("gamma_min", gamma_min_, 50);
 
+    node_.param("rotate_code", rotate_code_, -1);
```

Comment:
I think it's better to make the values readable.
e.g.
- make it `int` and check the value is one of `90`, `-90`, `180`
- make it string and allow `90CW`, `90CCW`, `180`